src/bci_build/package/helpers.py

```diff
@@ -25,3 +28,10 @@ def generate_package_version_check(
 
     return f"""# sanity check that the version from the tag is equal to the version of {pkg_name} that we expect
 {DOCKERFILE_RUN} [ "$(rpm -q --qf '%{{version}}' {pkg_name} | cut -d '.' -f -{cut_count})" = "{pkg_version}" ]"""
+
+
+def generate_from_image(os_version, container_name) -> str:
```

Comment:
I would also suggest to rename this function to something along the lines `from_image_tag` or `generate_from_image_url` or 

```suggestion
def generate_from_image(os_version: OsVersion, container_name: str) -> str:
    """Return the build tag or url of the container with the name `container_name` for the specified `os_version`."""
```